.github/workflows/ci.yml

```diff
@@ -0,0 +1,137 @@
+name: MO4 Coding Standard CI
+on:
+  pull_request:
+  push:
+  schedule:
+  - cron: '30 5 * * 1'
+jobs:
+  style-checks:
+    env:
+      XMLLINT_INDENT: "    "
+    runs-on: ubuntu-latest
+    steps:
+      - name: install dependencies
+        run: sudo apt update -qq && sudo apt -y install libxml2-utils
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          coverage: 'none'
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: '~/.cache/composer'
+          key: "cache-composer-${{ hashFiles('composer.json') }}"
+          restore-keys: 'cache-composer-'
+      - name: run composer
+        run: composer update --prefer-lowest --prefer-dist --no-interaction --no-progress
+      - run: xmllint --noout --schema vendor/squizlabs/php_codesniffer/phpcs.xsd MO4/ruleset.xml
+      - run: xmllint --noout --schema vendor/squizlabs/php_codesniffer/phpcs.xsd phpcs.mo4.xml
+      - run: xmllint --noout --schema vendor/squizlabs/php_codesniffer/phpcs.xsd phpcs.xml.dist
+      - run: diff -B MO4/ruleset.xml <(xmllint --format MO4/ruleset.xml)
+      - run: diff -B phpcs.mo4.xml <(xmllint --format phpcs.mo4.xml)
+      - run: diff -B phpcs.xml.dist <(xmllint --format phpcs.xml.dist)
+      - name: Stylecheck against MO4 itself
+        run: vendor/bin/phpcs
+  run-tests:
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
```

Comment:
why not this?
```suggestion
        os: [ubuntu-latest]
```

Reply:
Let's give it a try.. I picked the recommend version from the support matrix of `shivammathur/setup-php@v2` but the documented supported version of Windows also were not ... correct